docs/DeveloperGuide.md

```diff
@@ -520,7 +536,8 @@ testers are expected to do more *exploratory* testing.
    2. Test case: `archive 1`<br>
       Expected: 
       First client is archived, and is no longer shown in the active list. Details of the archived client shown in the status message.
-      Upon listing all archived clients (see [List Archived Clients](#listing-all-archived-clients)), the archived client can be seen.
+      Upon listing all archived clients (see [List Archived Clients](#listing-all-archived-clients)), the archived client can be seen.  
+      The color of the archived client will be darkened to indicator that the client has been archived.     
```

Comment:
Maybe we can have a diagram here of both lightened and darkened? Can ask them to refer to the previous diagram to avoid duplication